x/profiles/client/rest/tx.go

```diff
@@ -16,6 +16,8 @@ func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/profiles/{address}", deleteProfileHandler(cliCtx)).Methods("DELETE")
 	r.HandleFunc("/dtag-transfer-requests", requestDTagTransferHandler(cliCtx)).Methods("POST")
 	r.HandleFunc("/dtag-transfer-requests/accept", acceptTransferRequestHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/dtag-transfer-requests/refuse", refuseDTagTransferRequestHandler(cliCtx)).Methods("POST")
+	r.HandleFunc("/dtag-transfer-requests/cancel", cancelDTagTransferRequestHandler(cliCtx)).Methods("POST")
```

Comment:
I think that these endpoints should be changed to make sure they comply with the REST best practice of avoiding verbs:

```suggestion
	r.HandleFunc("/dtag-transfer-requests/{senderAddress}", acceptTransferRequestHandler(cliCtx)).Methods("POST")
	r.HandleFunc("/dtag-transfer-requests/{senderAddress}", refuseDTagTransferRequestHandler(cliCtx)).Methods("DELETE")
	r.HandleFunc("/dtag-transfer-requests/{recipientAddress}", cancelDTagTransferRequestHandler(cliCtx)).Methods("DELETE")
```

These could also simply the request body by removing the address (since it's already) present inside the endpoint itself.